examples/garrawarla/configs/config.yaml

```diff
@@ -0,0 +1,30 @@
+config:
+  build_stage:
+#    - $tempdir/$user/spack-stage
+    - $spack/var/spack/stage
```

Comment:
for these 4 service directories, consider adding `$USER` in the path, similar to what we're doing on Setonix, as it seemed more robust, e.g.:
```
$spack/var/spack/$USER/stage
```